core/src/test/java/com/redhat/lightblue/client/expression/query/ValueQueryTest.java

```diff
@@ -33,4 +35,25 @@ public void testToStringCallsToJson() throws JSONException {
 
         JSONAssert.assertEquals(expression.toString(), expression.toJson(), false);
     }
+
+    @Test
+    public void testValueWithWhiteSpace() throws JSONException {
+        ValueQuery expression = new ValueQuery("field1 = Red Hat Enterprise Linux");
+
+        String expectedJson = "{\"field\":\"field1\",\"op\":\"=\",\"rvalue\":\"Red Hat Enterprise Linux\"}";
+
+        JSONAssert.assertEquals(expectedJson, expression.toJson(), false);
+    }
+
+    @Test(expected=IllegalArgumentException.class)
+    public void testInvalidExpression() {
+        new ValueQuery("fie ld1 = Red Hat Enterprise Linux");
+        Assert.fail();
+    }
```

Comment:
I assume this is suppose to fail because there can't be a space in the field name? If so, would you mind dropping a comment above this unit test just to make sure that is clear.